tools/quickrm/main.go

```diff
@@ -94,7 +94,7 @@ func main() {
 		}
 	}
 	if failed {
-		os.Exit(1)
```

Comment:
I'd rather not change this since it looks like an internal thing only used for the build system.